src/canmatrix/canmatrix.py

```diff
@@ -517,10 +596,12 @@ def addSignalGroup(self, Name, Id, signalNames):
             if signalId is not None:
                 newGroup.addSignal(signalId)
 
-    def signalGroupbyName(self, name):
+    def signalGroupbyName(self, name):  # capital B: ByName?
```

Comment:
I would say yes.

Though I would `snake_case` anyways.  That'd be a separate issue I presume.

Reply:
It might be hard job, the refactoring capabilities of pycharm aren't unlimited as I saw :-( But yes, the fingers want to write snake case.